shub/settings.py

```diff
@@ -668,4 +668,20 @@ def __iter__(self):
     }
 
 if ENABLE_GITLAB_AUTH:  # noqa
-    SOCIAL_AUTH_GITLAB_SCOPE = ["api", "read_user"]
+    if "SOCIAL_AUTH_GITLAB_SCOPE" not in locals():
```

Comment:
Looks like this change from the other PR snuck in!

Reply:
i mentioned that is relies on changes from the #435 because the scripts need that other script to work properly.